src/components/ChatLog.js

```diff
@@ -0,0 +1,35 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = ({entries, toggleLiked}) => {
+    const chatLogContent = entries.map((message) => {
+        return (
+            <ChatEntry 
+            key={message.id}
+            id={message.id} 
+            sender={message.sender} 
+            body={message.body} 
+            timeStamp={message.timeStamp} 
+            liked={message.liked}
+            toggleLiked={toggleLiked} />
+        );
+    });
+
+    return <main>
```

Comment:
The App component already has a `<main>` that surrounds the ChatLog component. Now the HTML looks like this:
```
<main>
    <main>
        entries
    </main>
</main>
```
There should be only one main per page. There should be a different element used in the ChatLog as the top-level element.